tests/network/hotplug.go

```diff
@@ -238,6 +238,31 @@ var _ = SIGDescribe("nic-hotplug", func() {
 			Entry("In place", decorators.InPlaceHotplugNICs, inPlace),
 			Entry("Migration based", decorators.MigrationBasedHotplugNICs, migrationBased),
 		)
+
+		DescribeTable("VMI status should report the MAC address specified in the VM template spec", func(plugMethod hotplugMethod) {
+			checks.SkipIfNoKubeMacPool()
```

Comment:
I do not think you need this dependency, it should be transparent.
The expectation is that after everything is stable (hotplug finished), the VM and VMI specs in regards to the mac address should be in sync, and the status showing the correct value.

If there is no KMP around, then it will be correct in an empty manner.

Reply:
> the VM and VMI specs in regards to the mac address should be in sync, and the status showing the correct value.

The test asserts the VMI iface MAC address is as specified in the VM spec.
When KMP not exist in the cluster, the test fails because the VMI get random MAC address but the VM spec has empty MAC address.

Reply:
> When KMP not exist in the cluster, the test fails because the VMI get random MAC address but the VM spec has empty MAC address.

VM & VMI specs should always be in sync. Status can have any mac when no mac is mentioned in the spec.

Reply:
> VM & VMI specs should always be in sync. Status can have any mac when no mac is mentioned in the spec.

Right, I am trying to say that due to that behavior, the dependency on KMP still remains for this test.

Reply:
Well, I do not see it.